### ISSUE TITLE
Fail to install ignition on Fedora CoreOS: found busy partitions

### ISSUE THREAD

**Issue by balioune** [coreos/fedora-coreos-tracker#1573]
Hi,
I'm trying to install OKD on Openstack servers.
I used the Fedora CoreOS openstack's qcow2 image to deploy servers and I generate ignitions from the install-config.yaml.

I try to deploy the bootstrap using
```

## Reproduction Steps ##
```
 sudo coreos-installer install /dev/vda --ignition-url=http://INTERNAL_IP/okd4/bootstrap.ign --insecure-ignition --stream-base-url=http://INTERNAL_IP/ --image-url=http://INTERNAL_IP/images/fedora-coreos-38.20230819.3.0-openstack.ppc64le.qcow2.xz --copy-network
```


# Bug #
```

Partitions in use on /dev/vda:
    /dev/vda3 mounted on /boot
    /dev/vda4 mounted on /sysroot
Error: checking for exclusive access to /dev/vda

Caused by:
    found busy partitions
```

## coreos-installer Version ##
```
$ coreos-installer --version
coreos-installer 0.17.0
```

## Reproduction Steps ##
```
 sudo coreos-installer install /dev/vda --ignition-url=http://INTERNAL_IP/okd4/bootstrap.ign --insecure-ignition --stream-base-url=http://INTERNAL_IP/ --image-url=http://INTERNAL_IP/images/fedora-coreos-38.20230819.3.0-openstack.ppc64le.qcow2.xz --copy-network
```

## Other Information ##
```
[core@host-10-253-235-58 ~]$ sudo gdisk -l /dev/vda
GPT fdisk (gdisk) version 1.0.9

Partition table scan:
  MBR: protective
  BSD: not present
  APM: not present
  GPT: present

Found valid GPT with protective MBR; using GPT.
Disk /dev/vda: 209715200 sectors, 100.0 GiB
Sector size (logical/physical): 512/512 bytes
Disk identifier (GUID): 8DD62A9F-F590-4E22-B8C7-D617E0D06B59
Partition table holds up to 128 entries
Main partition table begins at sector 2 and ends at sector 33
First usable sector is 34, last usable sector is 209715166
Partitions will be aligned on 2048-sector boundaries
Total free space is 2014 sectors (1007.0 KiB)

Number  Start (sector)    End (sector)  Size       Code  Name
   1            2048            4095   1024.0 KiB  EF02  BIOS-BOOT
   2            4096          264191   127.0 MiB   EF00  EFI-SYSTEM
   3          264192         1050623   384.0 MiB   8300  boot
   4         1050624       209715166   99.5 GiB    8300  root
[core@host-10-253-235-58 ~]$ sudo gdisk -l /dev/vdb
GPT fdisk (gdisk) version 1.0.9

Partition table scan:
  MBR: not present
  BSD: not present
  APM: not present
  GPT: not present

Creating new GPT entries in memory.
Disk /dev/vdb: 209715200 sectors, 100.0 GiB
Sector size (logical/physical): 512/512 bytes
Disk identifier (GUID): 651AC5AE-591B-41DB-819F-7AAD95313E26
Partition table holds up to 128 entries
Main partition table begins at sector 2 and ends at sector 33
First usable sector is 34, last usable sector is 209715166
Partitions will be aligned on 2048-sector boundaries
Total free space is 209715133 sectors (100.0 GiB)

Number  Start (sector)    End (sector)  Size       Code  Name
```

**Comment by travier** [comment]
Moving this issue to the tracker.

**Comment by balioune** [comment]
Thanks @travier, just for information do you known how to configure butane file ti run ignition at first boot ?

**Comment by dustymabe** [comment]
> From what you've written here, it looks like you're trying to install the bootstrap image from a booted OpenStack instance into itself, which will not work. You should generate the Ignition configs and then boot another OpenStack instance passing the bootstrap config via userdata.

Right. To make this more clear: @balioune you shouldn't have to run `coreos-installer` here because you have an OpenStack FCOS image in the Openstack cluster already. Just boot the FCOS image and provide it the `bootstrap.ign` in its userdata.

**Comment by dustymabe** [comment]
@balioune do you have any new information?

Should we close this out?
